app/models.py

```diff
@@ -43,6 +43,22 @@ def insert_roles():
     def __repr__(self):
         return '<Role \'%s\'>' % self.name
 
+class IdlingIncident(db.Model):
+    __tablename__ = "incidents"
+    vehicle_id = db.Column(db.Integer, primary_key=True)
```

Comment:
I think it's still valuable to have both (i.e. include another `license_plate` field).

1. This is just from me scrolling through `poll244.csv`  but there are many rows where they found the vehicle id but not license plate, e.g. rows 145, 757 to 759, 763 to 766. 
2. We should ask them about it; I'd imagine otherwise they wouldn't have asked in the past. Also above situation happens. 
3. I don't think CAC currently asks for whether it's government vehicle, and I don't think vehicle id is only for government ones (almost all observation rows have both).